ocl/orgs/urls.py

```diff
@@ -13,4 +13,5 @@
     url(r'^(?P<org>' + NAMESPACE_PATTERN + ')/members/(?P<user>' + NAMESPACE_PATTERN + ')/$', OrganizationMemberView.as_view(), name='organization-member-detail'),
     url(r'^(?P<org>' + NAMESPACE_PATTERN + ')/sources/', include('sources.urls')),
     url(r'^(?P<org>' + NAMESPACE_PATTERN + ')/collections/', include('collection.urls')),
+    url(r'^(?P<org>' + NAMESPACE_PATTERN + ')/openmrsdictionaries/', include('openmrsdictionary.urls')),
 )
```

Comment:
Could you add a new line at the end of the file

Reply:
resolved